shared/src/main/scala/amf/Core.scala

```diff
@@ -16,7 +16,6 @@ import amf.core.unsafe.PlatformSecrets
 import scala.concurrent.ExecutionContext
 import scala.scalajs.js.annotation.JSExportAll
 
-@JSExportAll
```

Comment:
The CoreWrapper class in amf is the one that exports this behaviour. A Scala Object cannot be exported without the `@JSTopLevelExport` annotation